mozilla-release/toolkit/mozapps/extensions/internal/XPIProvider.jsm

```diff
@@ -7953,6 +7955,7 @@ Object.assign(SystemAddonInstallLocation.prototype, {
       logger.warn(`System add-on ${aAddon.id} isn't restartless.`);
       return false;
     }
+    */
```

Comment:
@chrmod did you check what are the consequences of these changes.
Why did Mozilla disallow non-bootstrapped packed addons?